Automattic/wp-calypso
# Add/new email provider comparison component

#### Changes proposed in this Pull Request

This is mostly new additions to the [previous PR](https://github.com/Automattic/wp-calypso/pull/58632), that creates a react component for ProfessionalEmail
It adds ability to create a mailbox and it also rewrites some types. Lastly it adds the new Professional Email SVG to the list of features.

### Work in progress
![image](https://user-images.githubusercontent.com/5689927/144054184-f23f2a08-06f2-4e5f-9127-3fd35606a2c6.png)

#### Testing instructions

1. Go to a site with domains without email subscription
2. Click on 'Add email' for the desired site
3. Once you are in the comparison page, add at the end the following flag: `?flags=emails/new-email-comparison`
4. You should be able to add the professional email item in your cart

**Remarks**: visible through the path /email/{domain}/purchase/{site}?flags=emails/new-email-comparison

You should see the beginning of a new component being developed.

Related to 1200182182542585-as-1201414571816235
